server/historian/packages/historian-base/src/runnerFactory.ts

```diff
@@ -45,6 +45,7 @@ export class HistorianResourcesFactory implements core.IResourcesFactory<Histori
 		customizations: IHistorianResourcesCustomizations,
 	): Promise<HistorianResources> {
 		const redisConfig = config.get("redis");
+		const redisTTLOverride = config.get("redisTTLOverride");
```

Comment:
Why we need an override but not able to config a value for historian own? Is it because the config is shared across all component? If so, I feel it is more manageable by change configs to say `historianRedisConfig`, `gitrestRedisConfig` sort of things